docs/api-publishing/02-Components.md

```diff
@@ -52,6 +52,10 @@ API consumer application developers will expect to be able to follow a simple an
 
 API Providers **MUST** provide a developer onboarding function that enables API Consumer developers to create and manage their client application credentials and request application level access to API Provider APIs.
 
+<ApiStandard id="HNZAS_MUST_have_compliance" type="MUST">API providers **MUST** provide compliance tests that API consumers are required to pass in order to onboard to In Confidence or higher-level APIs.</ApiStandard>
```

Comment:
Should this perhaps be slightly re-worded to say something along the lines of `MUST provide minimum compliance requirements to API Consumers` and `artefacts SHOULD be available via a web experience`?

Reply:
Included in the PR. Pls review.